examples/all-clusters-app/nrfconnect/boards/nrf7002dk_nrf5340_cpuapp.overlay

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
```

Comment:
The copyright header is not correct here and in several other places. It should be the one used by the Project CHIP authors.

Reply:
> Shouldn't we also add Wi-Fi support to the lighting-app?

I guess, not yet, we align to the NCS 2.1.1.

Reply:
Yes and no :) We align to the NCS 2.1.1, but next upmerge will be done from Matter upstream to NCS main, that has lighting-app WiFi support. So in such case there will be problem with upmerge, because samples will not be aligned. I mean it's ok for me to not add it now, but I think functionality should not conflict with NCS 2.1.1 code, right?

Reply:
hmm, ok agree, it can speed up next upmerge. I will add it too.